<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_analyse">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="layout_analyse">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QProgressBar" name="progr_Analyse">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Resultat :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_ResAnalyse">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="bouton_LancerAnalyse">
             <property name="text">
              <string>Analyser</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="check_accent">
             <property name="toolTip">
              <string>Convertit les &quot;é&quot; en &quot;e&quot;, au lieu de faire comme s'il n'existait pas.
 C'est mal fait donc très long, et ça sert à rien sur les listes de mots sans accents</string>
             </property>
             <property name="text">
              <string>Gestion des accents</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="1" colspan="2">
            <widget class="QPushButton" name="bouton_selecFichier">
             <property name="text">
              <string>(Aucun)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radio_LanPerso">
             <property name="text">
              <string>Choisir une liste de mot</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="label_Analyse">
             <property name="text">
              <string>Etape 1 : Analyse d'une langue</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QRadioButton" name="radio_LangDef">
             <property name="text">
              <string>Utiliser la langue par défaut (français sans accent)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_generation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="layout_generation">
         <item row="4" column="1">
          <widget class="QSpinBox" name="spin_tailleMax">
           <property name="suffix">
            <string> lettres</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="bouton_clear">
           <property name="text">
            <string>Nettoyer</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label_Generation">
           <property name="text">
            <string>Etape 2 : Génération de mots</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Taille max des mots</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" rowspan="6">
          <widget class="QTextEdit" name="text_ResMots">
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_NbMotGen">
             <property name="text">
              <string>Nombre de mots à générer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_NbMots">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="bouton_GenMots">
           <property name="text">
            <string>Inventer des mots</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="check_forceTaille">
           <property name="text">
            <string>Forcer la taille des mots</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
